### PR TITLE
Adding force to the RemoveNode API

### PR DESCRIPTION
**Pull request by diogomonica**
Dependent on: https://github.com/docker/swarmkit/pull/1253

Allows us to force-remove a node that is active.

Signed-off-by: Diogo Monica <diogo.monica@gmail.com>

### CODE REVIEW

**Comment by aaronlehmann** [comment]
LGTM (not a maintainer)

**Comment by vieux** [comment]
LGTM

**Comment by diogomonica** [comment]
@vieux is this worthy of bumping the API version? And can we do it in a point-release?

**Comment by vieux** [comment]
I think we can add it in a point-release /cc @tiborvass ?

File: client/interface.go [hunk 1]
```diff
@@ -94,7 +94,7 @@ type NetworkAPIClient interface {
 type NodeAPIClient interface {
 	NodeInspectWithRaw(ctx context.Context, nodeID string) (swarm.Node, []byte, error)
 	NodeList(ctx context.Context, options types.NodeListOptions) ([]swarm.Node, error)
-	NodeRemove(ctx context.Context, nodeID string) error
+	NodeRemove(ctx context.Context, nodeID string, force bool) error
```
Comment (stevvooe) [inline client/interface.go]: Create an options struct. It's easier to read if you don't know the API:

```go
client.NodeRemove(ctx, id, types.NodeRemoveOptions{Force: true})
```

Comment (diogomonica) [inline client/interface.go]: Seems like overkill, but sure.

**Comment by stevvooe** [comment]
LGTM

**Comment by vdemeester** [comment]
LGTM 🐯

**Pull request merged**